jobHandler/injected/run.py

```
@@ -0,0 +1,200 @@
+from pathlib import Path
+from clp_logging.handlers import CLPFileHandler
+clp_handler = CLPFileHandler(Path('./run.clp.zst'))
+logger = logging.getLogger('adli')
+logger.setLevel(logging.INFO)
+logger.addHandler(clp_handler)
+
+def aspAdliVarLog(val, varid):
+    try:
+        val = json.dumps(val, default=lambda o: o.__dict__)
+    except:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)